### PR TITLE
Fixing a formula for multinomial regression on GLM docs

### PR DESCRIPTION
**Pull request by MikolajBak**
Added a correct formula to fit page 18 of https://www.h2o.ai/wp-content/uploads/2018/01/GLM-BOOKLET.pdf

### CODE REVIEW

**Comment by michalkurka** [comment]
@MikolajBak thank you for your contribution

@wendycwong can you please take a look and approve?

**Comment by wendycwong** [comment]
@michalkurka Yes, will do

File: h2o-docs/src/product/data-science/glm.rst
Comment (hannah-tillman) [inline h2o-docs/src/product/data-science/glm.rst:427]: Hello! You need to indent the math code right here for it to render properly please and thank you :)

Comment (MikolajBak) [reply h2o-docs/src/product/data-science/glm.rst]: Hi, thanks for the comment, I missed this. It should be fixed now :)

**Review by wendycwong — APPROVED** [review]
Thank you for adding the formula.  Wendy

**Comment by wendycwong** [comment]
Please wait for @hannah-tillman's approval before merging.

**Review by hannah-tillman — APPROVED** [review]

**Pull request merged**
